Week3/homework/README_Homework.md

```diff
@@ -0,0 +1,78 @@
+# Databases Week3 Homework - Ammar Vehbi Merakli
+
+## Raw body to be used in postman as JSON (application/json):
+
+```
+{
+"user": "xxx",
+"email": "xxx@gmail.com",
+"todoListName": "xxx",
+"reminder": "dd/mm/yyyy",
+"todo": "xxx",
+"done": "false",
+"dueDate": "dd/mm/yyyy",
+"tag": "xxx"
+}
+```
+
+## Follow the steps to use functions properly:
+
+1 - Use the dump file as source from mysql prompt to make your entire database ready to be used:
+`source path/to/the/dump/file/ToDoApp_dump.sql`
+
+2 - Run the application with the command `node index.js`
+
+3 - You should run the functions by
+
+    - selecting proper method
+
+    - using proper url
+
+    - inputting necessary information if needed at body in postman
+
+## Functions:
+
+**Users Part:**
+
+    - To create a new user (POST):
+    `http://localhost:3001/ToDoApp/createNewUser`
```

Comment:
**Non-blocking comment:** 

when you describe PUT/POST/PATCH requests - it would be great to specify also what kind of body parameters it expects: names, type of values (like string, int, bool), length, etc.

There is a tool for specifying that: https://swagger.io

**I'm saying this because of**, for instance, you'll be a backend developer, and you'll build an API. Then you'll have a lot of talks with frontend-guys, who will be asking you about how your API works, what does it expect and other stuff like this.

But if you use swagger - you'll say: here is swagger file, everything is described there. Useful.

Reply:
Since my internet signal is very poor I couldn't look into deeply swagger, bu definitely I will. For now I updated my `README_Homework.md` file. And also I will try to push my swagger file ASAP. Thanks a lot @ibudasov